<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainScreen</class>
 <widget class="QMainWindow" name="mainScreen">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>同步相量测量装置监控</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/config/image/Main.png</normaloff>:/config/image/Main.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="meaSure" name="WidgetMeasure">
       <attribute name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/config/image/Measure.png</normaloff>:/config/image/Measure.png</iconset>
       </attribute>
       <attribute name="title">
        <string>测量</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
      <widget class="Sync" name="WidgetSync">
       <attribute name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/config/image/Sync.png</normaloff>:/config/image/Sync.png</iconset>
       </attribute>
       <attribute name="title">
        <string>同步</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="Dev1"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Dev1">
   <property name="text">
    <string>设备一</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>meaSure</class>
   <extends>QWidget</extends>
   <header>measure.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Sync</class>
   <extends>QWidget</extends>
   <header location="global">sync.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
